README/ReleaseNotes/v628/index.md

```diff
@@ -182,12 +182,42 @@ RNTuple is still experimental and is scheduled to become production grade in 202
 ### New features
 
 - Add [`GraphAsymmErrors`](https://root.cern/doc/master/classROOT_1_1RDF_1_1RInterface.html#acea30792eef607489d498bf6547a00a6) action that fills a TGraphAsymmErrors object.
+- Introduce [`RDatasetSpec`](https://root.cern/doc/master/classROOT_1_1RDF_1_1Experimental_1_1RDatasetSpec.html) as an
+experimental class to specify the input dataset to an RDataFrame.
+- Introduce metadata handling in the specification of the input dataset. An RDataFrame can also be created directly from
+a specification (including metadata) in JSON format.
 
 ### Notable bug fixes and improvements
 
 - Fix the node counter of [`SaveGraph`](https://root.cern/doc/master/namespaceROOT_1_1RDF.html#ac06a36e745255fb8744b1e0a563074c9), where previously `cling` was getting wrong static initialization.
 - Fix [`Graph`](https://root.cern/doc/master/classROOT_1_1RDF_1_1RInterface.html#a1ca9a94bece4767cac82968910afa02e) action (that fills a TGraph object) to properly handle containers and non-container types.
 - The [`RCsvDS`](https://root.cern.ch/doc/master/classROOT_1_1RDF_1_1RCsvDS.html) class now allows users to specify column types, and can properly read empty entries of csv files.
+- Fixed a bug where the `Display` operation would not show the correct amount of entries requested by the user if called
+together with other operations ([PR](https://github.com/root-project/root/pull/11398)).
+- `Stats` can now be used in conjunction with `Vary`.
+
```

Comment:
```suggestion
- factory functions for RDataFrames reading CSV files, RNTuples, Arrow tables, etc. have been renamed in order to increase consistency, e.g. `MakeCsvDataFrame` is now `FromCSV`. The old wording is still available but deprecated.
- the precision of `Sum`s and `Mean`s of single-precision floating point values has been greatly improved by employing Kahan summations
- the content of RDF's [execution logs](https://root.cern/doc/master/classROOT_1_1RDataFrame.html#rdf-logging) has been streamlined in order to make them more useful
```

Reply:
I didn't include `FromCSV` and friends in this list because it's already in the `Deprecation and Removal` part of the relnotes, but in fact I agree it's better to have it here as well.